tsconfig.json

```diff
@@ -1,11 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "lib": ["dom", "dom.iterable", "esnext"],
```

Comment:
I feel like I'd prefer new lines in a config file. No strong feeling here though.

Reply:
I'm not seeing options for this.